samples/word/99-preview-apis/manage-comments.yaml

```diff
@@ -0,0 +1,235 @@
+id: word-manage-comments
+name: Manage comments
+description: Basic operations using comments.
```

Comment:
Can we elaborate on this a bit? Any keywords here drive search, so maybe something enumerating the operations?

Reply:
Will do